### DISCUSSION TITLE
Publishing @actionsflow/trigger-activitypub

### DISCUSSION THREAD

**Discussion by rg-wood** [actionsflow/actionsflow#45] [Q&A] [answered]
I'm attempting to publish my new ActivityPub trigger [@actionsflow/trigger-activitypub](https://github.com/rg-wood/actionsflow-trigger-activitypub), but I keep getting a 404 when I attempt to.

```
npm ERR! code E404
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/@actionsflow%2ftrigger-activitypub - Not found
npm ERR! 404
npm ERR! 404  '@actionsflow/trigger-activitypub@1.0.2' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

I think this is because I don't have access to the `actionsflow` scope/organisation. Would it be possible to get an invite? 

My profile is here:

https://www.npmjs.com/~rgladwell

**Comment by theowenyoung** [comment, marked as answer]
Hi, I've invited you on the npm.

**Reply by rg-wood** [reply]
Thanks!
